src/main/java/com/dataloom/organizations/controllers/OrganizationsController.java

```diff
@@ -81,13 +86,14 @@ public Void destroyOrganization( @PathVariable( ID ) UUID organizationId ) {
         List<UUID> aclKey = ensureOwner( organizationId );
         authorizations.deletePermissions( aclKey );
         organizations.destroyOrganization( organizationId );
+        securableObjectTypes.deleteSecurableObjectType( ImmutableList.of( organizationId ) );
         return null;
     }
 
     @Override
     @PutMapping(
         value = ID_PATH + TITLE,
-        consumes = MediaType.TEXT_PLAIN_VALUE )
+        consumes = MediaType.APPLICATION_JSON_VALUE )
```

Comment:
`@RequestBody String title` is still only a `String`, not a JSON object, so does this need to change to `APPLICATION_JSON_VALUE`?